notes/zrq/20210122-02-ansible-deploy.txt

```diff
@@ -109,10 +103,18 @@ EOF
 
     >   ....
     >   ....
+    >   ....
+    >   ....
+    >   PLAY [Mount volumes on workers] *************************************************************************************************************************************************************************************************************
+    >   
+    >   TASK [Check btrfs tools are installed] ******************************************************************************************************************************************************************************************************
+    >   fatal: [worker03]: FAILED! => {"changed": false, "msg": "Failed to synchronize cache for repo 'fedora-modular'", "rc": 1, "results": []}
```

Comment:
Is this just a temporary issue accessing one of the fedora cache sites? Does worker03 still get deployed with a healthy state?

Reply:
I think this is going to be a problem until I update our VMs to using a more recent version of Fedora.
I'm working on a short term fix and hope to include it in the next PR.